src/Core/Content/Product/SalesChannel/Listing/Filter/ManufacturerListingFilterHandler.php

```diff
@@ -13,14 +13,16 @@
 #[Package('inventory')]
 class ManufacturerListingFilterHandler extends AbstractListingFilterHandler
 {
+    final public const FILTER_ENABLED_REQUEST_PARAM = 'manufacturer-filter';
```

Comment:
would be cool, if you could use those new constants also in the tests

Reply:
@mitelg yes I can do that, although I am always not sure if it is better or not to use the constants in the tests, as this would not catch a changed constant. But if that is the preferred way, I will adjust the PR accordingly.

Reply:
good point :thinking: leave it for now :slightly_smiling_face: